src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/InputSuppressionXpathFilterEscapeString.java

```diff
@@ -0,0 +1,28 @@
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;
+
+/*
+ * Config: Default.
+ */
+public class InputSuppressionXpathFilterEscapeString {
+
+    String quoteChar = "\"escaped\"" +
+            "2"; // violation
```

Comment:
please use IllegalTokenCheck for this input to simplify input file.

Reply:
Done